test/AWS.Deploy.CLI.IntegrationTests/AWS.Deploy.CLI.IntegrationTests.csproj

```diff
@@ -19,6 +19,7 @@
     <ProjectReference Include="..\..\src\AWS.Deploy.Common\AWS.Deploy.Common.csproj" />
     <ProjectReference Include="..\..\src\AWS.Deploy.ServerMode.Client\AWS.Deploy.ServerMode.Client.csproj" />
     <ProjectReference Include="..\AWS.Deploy.CLI.Common.UnitTests\AWS.Deploy.CLI.Common.UnitTests.csproj" />
+    <ProjectReference Include="..\AWS.Deploy.CLI.UnitTests\AWS.Deploy.CLI.UnitTests.csproj" />
```

Comment:
Integration tests must not be related to Unit Tests. We have a common project to share responsibilities.